libraries/adaptive-expressions/src/expressionProperties/numberExpression.ts

```diff
@@ -14,7 +14,11 @@ import { Expression } from '../expression';
  * String values are always interpreted as an expression, whether it has '=' prefix or not.
  */
 export class NumberExpression extends ExpressionProperty<number> {
+    /**
+     * Initializes a new instance of the `ArrayExpression<T>` class.
+     * @param value A `float` or `string` expression which resolves to a `float`.
```

Comment:
Should we use `float` and `int` in here? they are all `number`.

Reply:
I corrected it to "int `number`" and "float `number`" since those descriptors are relevant for the to describe JSON conversion units.